docs/topic/billing/accounts.md

```diff
@@ -0,0 +1,155 @@
+
+We will not dig too deep into it, lest we wake some demons. However, since
+we are primarily concerned with billing, understanding the following concepts
+is more than enough.
+
+1. An [AWS Account](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_getting-started_concepts.html#account)
+   contains all the cloud resources one may create.
+2. An [AWS Organization](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html)
+   is a way to *group* multiple AWS accounts together. One AWS Account in
+   any AWS Organization is deemed the [management account](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_getting-started_concepts.html#account).
+   The management account sets up access control, billing access, billing
+   data export, etc for the entire organization.
+
+Each AWS Account can have billing attached in one of two ways:
+
+1. Directly attached to the account, via a credit card or credits
+2. If the account is part of an AWS Organization, it can use the billing
+   information of the *Management Account* of the organization it is a
+   part of. This is known as [consolidated billing](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/consolidated-billing.html)
+
+So billing information is *always* associated with an AWS Account,
+*not* AWS Organization. This is a historical artifact - AWS Organizations
+were only announced [in 2016](https://aws.amazon.com/about-aws/whats-new/2016/11/announcing-aws-organizations-now-in-preview/),
+a good 9 years after AWS started. So if this feels a little hacked on,
+it is because it is!
+
+A pattern many organizations (including 2i2c) follow is to have an AWS Account
+that is completely empty and unused, except for being designated as the
+management account. Billing information (and credits) for the entire
+organization is attached to this account. This makes access control much
+simpler, as the only people who get access to this AWS account are those
+who need to handle billing.
+
+## Azure
+
+We currently have no experience or knowledge here, as all our Azure
+customers handle billing themselves. This is not uncommon - most Azure
+users want to use it because they already have a pre-existing *strong*
+relationship with Microsoft, and their billing gets managed as part of
+that.
+
+## Billing reports in the web console
+
+Cloud providers provide a billing console accessible via the web, often
+with very helpful reports. While not automation friendly (see next section
+for more information on automation), this is very helpful for doing ad-hoc
+reporting as well as trying to optimize cloud costs. For dedicated clusters,
+this may also be directly accessible to hub champions, allowing them to
+explore their own usage.
+
+### GCP
+
+You can get a list of all billing accounts you have access to [here](https://console.cloud.google.com/billing)
```

Comment:
```suggestion
You can get a list of all billing accounts you have access to via the [billing console](https://console.cloud.google.com/billing)
```

For accessibility/usability reasons, it is not best practice to have hypertext for links that read something like "click here"